src/lib/types/WebManifest.ts

```diff
@@ -0,0 +1,36 @@
+
+/*
+ * Copyright 2019 Google Inc. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+export interface WebManifestIcon {
+  src: string;
+}
+
+export interface WebManifestShortcutJson {
+  name: string | undefined;
```

Comment:
You could make these optional, right?

```typescript
    name?: string;
    short_name?: string;
```

Reply:
Didn't actually know this was a thing. I wonder if there's a lint rule for that.

Reply:
done

Reply:
I couldn't find a lint rules for this, which actually makes sense, since there's a difference between both approaches. Using `string | undefined` actually forces you to assign to the variable in the construction, while optional doesn't.